.github/workflows/release-artifacts.yml

```diff
@@ -0,0 +1,77 @@
+name: Build and publish release packages
+permissions:
+  contents: write
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-linux-binaries:
+    strategy:
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
```

Comment:
Would it be possible to also add ` aarch64-unknown-linux-gnu` to the targets?

Reply:
Hi, I tried it.
I realised we can't use  https://github.com/taiki-e/upload-rust-binary-action action for aarch as this GA uses cross aarch container to compile code but we need to install few dependencies beforehand.

I tried to directly use cross by configuring Cross.toml but the build is still not going through- https://github.com/Shubham8287/qdrant/actions/runs/4787382241/workflow#L50, Also I am not able to build it locally with cross, dependencies still messing up after many attempts.

Can you please verify if you can build it locally for aarch as you understand dependencies much better?

Reply:
I asked you about `aarch64-unknown-linux-gnu` because I had trouble doing it in the past as well :smiley_cat: 

I propose to go with the PR as it is and work on `aarch64-unknown-linux-gnu` when it is needed.